src/main.rs

```diff
@@ -2,36 +2,18 @@ extern crate dice;
 
 fn run_app() -> Result<(), String> {
     use std::io;
-    use std::io::Write;
+    use dice::cli;
     use dice::wordlist;
 
-    println!(r#"Welcome to 🎲 Dice!
-v1.0.0-alpha
+    let output = io::stdout();
+    cli::welcome(output);
 
---------------------------------------------------------------------
+    let output = io::stdout();
```

Comment:
The `output` variable is [moved][doc] when printing the welcome message two lines above. I am not sure if creating a new variable here is the correct thing to do, but the error message when I don't makes reference to **copying**, so I guess it's fine? 

```
error[E0382]: use of moved value: `output`
  --> src/main.rs:12:32
   |
9  |     cli::welcome(output);
   |                  ------ value moved here
...
12 |     cli::prompt_for_user_input(output);
   |                                ^^^^^^ value used here after move
   |
   = note: move occurs because `output` has type `std::io::Stdout`, which does not implement the `Copy` trait
```

  [doc]: https://doc.rust-lang.org/book/second-edition/ch04-01-what-is-ownership.html#ways-variables-and-data-interact-move